### ISSUE TITLE
UnicodeDecodeError when using MoleculeDatabaseAggregator

### ISSUE THREAD

**Issue by assaron** [saezlab/lipyd#2]
Hi,

I'm trying to use your tool to annotate some of our lipidomics data, however I've stumbled upon an error, when I try to use MoleculeDatabaseAggregator.

Here are the commands I used:

```
from lipyd import moldb
from lipyd import name
from lipyd.lipproc import *
db = moldb.MoleculeDatabaseAggregator()
```

On the last command is starts successfully, but fails on indexing SwissLipids database. Here is the log:

```
        Downloading `export.php` from www.swisslipids.org -- 74.32MB downloaded: : 74.4Mit [02:51, 434kit/s]                                                                                                        
	:: Ready. Resulted `gz extracted data` of type file object.                                                                                   
	:: Local file at `/home/alserg/.lipyd/cache/5404f52defd16c3cfe05d4b40610f76e-export.php`.
	:: Loading data from cache previously downloaded from www.swisslipids.org
	:: Ready. Resulted `gz extracted data` of type file object.                                                                                   
	:: Local file at `/home/alserg/.lipyd/cache/5404f52defd16c3cfe05d4b40610f76e-export.php`.
        Indexing SwissLipids -- busy:  54%|███████████████████████████████████████████████████████████████████████▎                                                             | 365M/681M [00:09<00:08, 36.2Mit/s]---------------------------------------------------------------------------
UnicodeDecodeError                        Traceback (most recent call last)
<ipython-input-2-6b5c2ae0d558> in <module>()
----> 1 db = moldb.MoleculeDatabaseAggregator()

/home/alserg/.local/lib/python3.7/site-packages/lipyd/moldb.py in __init__(self, resources, tolerance, fa_args, sph_args, build, verbose, database_preference)
    785         if build:
    786 
--> 787             self.build()
    788 
    789 

/home/alserg/.local/lib/python3.7/site-packages/lipyd/moldb.py in build(self)
    836 
    837         self.init_rebuild()
--> 838         self.load_databases()
    839         self.auto_glycerophospholipids()
    840         self.auto_glycerolipids()

/home/alserg/.local/lib/python3.7/site-packages/lipyd/moldb.py in load_databases(self)
    852         for cls, resargs in self.resources.values():
    853 
--> 854             resource = cls(**resargs)
    855 
    856             self._mass_data.extend(resource)

/home/alserg/.local/lib/python3.7/site-packages/lipyd/moldb.py in __init__(self, levels, silent, nameproc_args, branched, exact_mass_formula_fallback)
    233         self.url = settings.get('swisslipids_url')
    234         self.load()
--> 235         self.make_index()
    236 
    237     def set_levels(self, levels):

/home/alserg/.local/lib/python3.7/site-packages/lipyd/moldb.py in make_index(self)
    324                     self.prg.step(len(l))
    325 
--> 326                 ll = l.decode('utf-8').split('\t')
    327 
    328                 if ll[1] in self.levels:

UnicodeDecodeError: 'utf-8' codec can't decode byte 0xdf in position 34: invalid continuation byte
```

**Comment by deeenes** [comment]
Hi,

Apparently SwissLipids switched to latin-1 encoding, I fixed it in ccee4e2. Also I updated the LipidMaps URL, the recent version of the database contains 1,100 more structures.

Best,

Denes

**Comment by assaron** [comment]
Thanks!
